keyboards/handwired/illera16/info.json

```
@@ -0,0 +1,48 @@
+{
+    "manufacturer": "victor_illera",
+    "keyboard_name": "illera16",
+    "maintainer": "villeramassana",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["D4", "C6", "D7", "E6"],
+        "rows": ["F4", "F5", "F6", "F7"]
+    },
+    "processor": "atmega32u4",
```

1. `development_board` supersedes `processor` and `bootloader`
2. encoder and rgblight settings migrated from `rules.mk` and `config.h`
```suggestion
    "development_board": "promicro",
    "diode_direction": "COL2ROW",
    "features": {
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": true,
        "encoder": true,
        "rgblight": true
    },
    "matrix_pins": {
        "cols": ["D4", "C6", "D7", "E6"],
        "rows": ["F4", "F5", "F6", "F7"]
    },
    "encoder": {
        "rotary": [
            {"pin_a": "B5", "pin_b": "B4", "resolution": 2},
            {"pin_a": "B6", "pin_b": "B2", "resolution": 2}
        ]
    },
    "rgblight": {
        "led_count": 6,
        "layers": {
            "enabled": true
        },
    },
    "ws2812": {
        "pin": "D0"
    },
```